### PR TITLE
Handle managed_policy_arns in aws_iam_roles

### PR DESCRIPTION
**Pull request by eliecharra**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | yes
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #627
| ❓ Documentation  | no

### CODE REVIEW

**Review by lotoussa — DISMISSED** [review]
ok i see how you do it, cool
lgtm

**Review by lotoussa — APPROVED** [review]

**Review by wbeuil — CHANGES_REQUESTED** [review]
Small changes

File: pkg/resource/aws/aws_iam_role_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+				Env: map[string]string{
+					"AWS_REGION": "us-east-1",
+				},
+				Check: func(result *test.ScanResult, stdout string, err error) {
+					if err != nil {
+						t.Fatal(err)
+					}
+					result.AssertDriftCountTotal(0)
+					result.AssertDeletedCount(0)
+					result.AssertManagedCount(1)
```
[the remark this answers is not in the document]
Comment (eliecharra) [reply pkg/resource/aws/aws_iam_role_test.go:26]: Here I do not want to catch unmanaged resources (they persist dangling role to allow access to driftctl in QA pipeline)

[the remark this answers is not in the document]
Comment (wbeuil) [reply pkg/resource/aws/aws_iam_role_test.go:26]: Ok got it !

**Review by wbeuil — APPROVED** [review]

**Pull request merged**